src/api/db/student_course_selection.py

```diff
@@ -0,0 +1,61 @@
+class student_course_selection:
+	def __init__(self, db_conn):
+		self.db_conn = db_conn
+
+	def add_selection(self, name, sem, uid, cid):
+		sql = 	"""
+				INSERT INTO
+					student_course_selection (user_id, semester, course_name, crn)
+				VALUES
+					(%s, %s, %s, %s)
+				"""
+		resp, error = self.db_conn.execute(sql, [uid, sem, name, cid], False)
+		return (True, None) if not error else (False, error)
+
+		
+
+	def remove_selection(self, name, sem, uid, cid):
+		if cid == '-1':
```

Comment:
is `cid` being -1 a way for the dev to specify to delete all courses? Feels a little obscure. Possible to add documentation or make cid a keyword argument?

Reply:
Can't use none if it is part of primary key. I would love to use a better value if you have any ideas

Reply:
Also, -1 is to specify it as the course object, instead of a specific section, it's not a specifier to delete all. It's just that if the course object is being deleted, all associated sections also have to be deleted

Reply:
I just mean you would change the condition `if cid == '-1'` to `if cid is None`. So that cid is an optional parameter.

Reply:
Done